src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractCheck.java

```diff
@@ -240,13 +240,33 @@ public final void setTabWidth(int tabWidth) {
     /**
      * Helper method to log a LocalizedMessage.
      *
+     * {@code CommonUtils.lengthExpandedTabs} returns column number considering tabulation
```

Comment:
As I stated in previous review, I think this comment should be an in-method comment.
From Javadoc view, there is no `CommonUtils.lengthExpandedTabs` in method definition, so this only makes sense if user reads source code too.

Reply:
@timurt Are we going to do this?

Reply:
done